ert_shared/models/ensemble_experiment.py

```diff
@@ -26,18 +25,9 @@ def runSimulations__(self, arguments, run_msg):
 
         self.setPhaseName(run_msg, indeterminate=False)
 
-        if FeatureToggling.is_enabled("ensemble-evaluator"):
-            ee_config = arguments["ee_config"]
-            num_successful_realizations = self.run_ensemble_evaluator(
-                run_context, ee_config
-            )
-        else:
-            self._job_queue = self._queue_config.create_job_queue()
-            num_successful_realizations = (
-                self.ert()
-                .getEnkfSimulationRunner()
-                .runEnsembleExperiment(self._job_queue, run_context)
```

Comment:
As we are removing the usage of `SimulationRunner` to a large degree here, we should investigate if we can further remove the implementation as well, or if it is in use anywhere.

Reply:
Everest depends on _some_ code in the sim. runner, IIRC.